backend/tests/v5/unit/models/users.test.js

```diff
@@ -72,7 +72,67 @@ const testGetFavourites = () => {
 	});
 };
 
+const testAppendFavourites = () => {
+	const favouritesData = {
+		teamspace1: ['a', 'b', 'c'],
+		teamspace2: ['d'],
+	};
+
+	describe('Add containers to favourites', () => {
+		test('Adding an array of containers of a new teamspace', async () => {
+			jest.spyOn(db, 'findOne').mockResolvedValue({ customData: { starredModels: favouritesData } });
+			await User.appendFavourites('xxx', 'teamspace3', ['e', 'f']);
+			const newFavourtiesData = favouritesData;
```

Comment:
this test is meaningless - you are literally testing if the same object equals to the same object.

The only output you need to test is that it doesn't fail. so you don't need these 3 lines

Reply:
same for the next 2 tests